assets/css/style.css

```diff
@@ -0,0 +1,715 @@
+@import "./reset.css";
```

Comment:
- [ ] Hello team, this project requires you to have a Mobile-first approach. Therefore you should not have media queries for small screens. I will paste the screenshot here.

> ![image](https://user-images.githubusercontent.com/57414789/82961198-44611780-9f82-11ea-8fc7-be2522f473e0.png)